msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/SignUpStates.kt

```diff
@@ -288,6 +297,25 @@ class SignUpCodeRequiredState internal constructor(
             }
         }
     }
+
+    override fun writeToParcel(parcel: Parcel, flags: Int) {
+        parcel.writeString(flowToken)
+        parcel.writeString(username)
+    }
+
+    override fun describeContents(): Int {
+        return 0
+    }
+
+    companion object CREATOR : Parcelable.Creator<SignUpCodeRequiredState> {
+        override fun createFromParcel(parcel: Parcel): SignUpCodeRequiredState {
+            return SignUpCodeRequiredState(parcel)
+        }
+
+        override fun newArray(size: Int): Array<SignUpCodeRequiredState?> {
```

Comment:
Why is this nullable?

Reply:
This is implementing the CREATOR interface and the signature comes from that interface.